src/KingSystem/Sound/sndMusicInfo.cpp

```diff
@@ -0,0 +1,117 @@
+#include "KingSystem/Sound/sndMusicInfo.h"
+
+namespace ksys::snd {
+
+MusicParam::ParamPerType::ParamPerType()
+    : mBPM(0x6E, "Bpm", "BPM", "Min=60,Max=180", this),
```

Comment:
decimal please (since this is clearly intended to be a BPM value, and BPMs are typically expressed in base 10)